src/__init__.py

```diff
@@ -3,19 +3,8 @@
 import subprocess
 from pathlib import Path
 
-from .do_api import (  # noqa: F401
```

Comment:
removing this because i want to explicitly import `auth` when we interact with the auth module

Reply:
No strong opinions